email.go

```diff
@@ -100,3 +101,27 @@ func (s *EmailService) SendBatch(emailRequests []*Email) ([]*EmailResponse, *Res
 
 	return email, response, nil
 }
+
+// SendBatchTemplated will build and execute request to send batch templated emails via the API.
+func (s *EmailService) SendBatchTemplated(emailRequests []*Email) ([]*EmailResponse, *Response, error) {
```

Comment:
Could we perhaps rename to something more in alignment with postmark?

`SendBatchWithTemplates` perhaps?